src/Twig/Extension/BoltExtension.php

```diff
@@ -24,19 +25,23 @@ class BoltExtension extends Extension implements GlobalsInterface
     private $config;
     /** @var PathsProxy */
     private $paths;
+    /** @var Query */
+    private $queryEngine;
 
     /**
      * Constructor.
      *
      * @param EntityManagerInterface $em
      * @param Config                 $config
      * @param PathsProxy             $paths
+     * @param Query                  $query
      */
-    public function __construct(EntityManagerInterface $em, Config $config, PathsProxy $paths)
+    public function __construct(EntityManagerInterface $em, Config $config, PathsProxy $paths, Query $query)
```

Comment:
So this might cause a problem. The entity manager passed in here is the lazy-loading version, but `Query` activates the real EM, and this class is constructed during registration.

I had to go do a bit of checking to make sure I wasn't off of a path then :laughing: